frontend/index.html

```diff
@@ -36,24 +46,70 @@
     print "🍦 big x"
 end
 </textarea
-        >
-      </div>
-      <div class="out">
-        <div class="canvas">
-          <canvas id="canvas"></canvas>
-        </div>
-        <div class="read hidden">
-          <label>input&gt;</label>
-          <textarea id="read" rows="1"></textarea>
-        </div>
-        <div class="input slider hidden">
-          <input type="range" id="sliderx" min="0" max="100" />
-          <input type="range" id="slidery" min="0" max="100" />
+          >
         </div>
         <div class="output">
-          <textarea id="output" disabled></textarea>
+          <div class="canvas">
+            <canvas id="canvas"></canvas>
+          </div>
+          <div class="read hidden">
+            <label>input&gt;</label>
+            <textarea id="read" rows="1"></textarea>
+          </div>
+          <div class="input slider hidden">
+            <input type="range" id="sliderx" min="0" max="100" />
+            <input type="range" id="slidery" min="0" max="100" />
+          </div>
+          <!-- prettier-ignore -->
+          <div class="console" id="console">x: 12
+x: 12
```

Comment:
Why is the output window pre-loaded with this? Shouldn't it be empty until "run" is pressed?

Regardless, if you do want it, "x: 12" is here twice but when you run the welcome program, it only prints it once.

Reply:
fixed, thanks for the comment.